lib/config.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+var Promise = require("bluebird");
+var Confidence = require("confidence");
+var Joi = require("joi");
+
+var configSettings;
+
+function Config() {
+	configSettings = {
+		bandwidth : {
```

Comment:
Configuration files tend to complicate CD patterns. I think environment variables with reasonable defaults preferred and fail fast behaviour when that's not possible is simpler to maintain and fits our deployment patterns better.